desktop/main.js

```diff
@@ -12,6 +12,7 @@ const CONST = require('../src/CONST').default;
 const Localize = require('../src/libs/Localize');
 
 const port = process.env.PORT || 8080;
+const {DESKTOP_SHORTCUT_ACCELERATOR} = CONST;
```

Comment:
I thought `CONST` is already imported on L11?

Reply:
Not importing here,just destructuing

Reply:
🤦